app/frontend/components/common/welcome.tsx

```diff
@@ -115,7 +115,4 @@ class Welcome extends Component<WelcomeProps, {dontShowAgainCheckbox: boolean}>
   }
 }
 
-export default connect(
-  null,
-  actions
-)(Welcome)
+export default connect(null, actions)(Welcome)
```

Comment:
I didn't change this line, somehow the "Husky" pre-commit hook "prettified" this line.